xarray/core/variable.py

```diff
@@ -1048,6 +1061,8 @@ def chunk(
         name: str = None,
         lock: bool = False,
         inline_array: bool = False,
+        manager: Literal["dask", "cubed"] = "dask",
+        spec: T_CubedSpec = None,
```

Comment:
maybe it would be better to add `from_array_kwargs` and `rechunk_kwargs`? That way we don't need to add in manager specific arguments to the generic interface here.

Reply:
Presumably both `manager` and `spec` would come under `from_array_kwargs`? In which case what are you suggesting would come under `rechunk_kwargs`? Every other keyword argument?